ccd-definition/CaseEvent.json

```diff
@@ -200,5 +200,18 @@
     "ShowSummary": "N",
     "ShowEventNotes": "N",
     "CallBackURLSubmittedEvent": "http://fpl-service:4000/callback/case-submission"
+  },
+  {
+    "LiveFrom": 42795,
+    "CaseTypeID": "Shared_Storage_DRAFTType",
+    "ID": "PDF",
+    "Name": "View attached PDF",
```

Comment:
Let's rename to something like `Attach submitted form PDF` or `Generate submitted form PDF` since that action generates the PDF, it has nothing in common with PDF.